tests/cover/test_setup_teardown.py

```diff
@@ -95,15 +95,14 @@ def test_calls_setup_and_teardown_on_failure():
 def test_still_tears_down_on_failed_reify():
     x = HasSetupAndTeardown()
     with pytest.raises(AttributeError):
-        with settings(perform_health_check=False):
-            x.fail_in_reify()
+        x.fail_in_reify()
```

Comment:
Have I missed something, or are we now running the same test twice?

Reply:
You've not missed anything, no, I did!